src/drop.test-d.ts

```diff
@@ -0,0 +1,54 @@
+import { drop } from "./drop";
+import { pipe } from "./pipe";
+
+describe("data first", () => {
+  it("works on regular inputs", () => {
+    expectTypeOf(drop([1, 2, 3, 4, 5], 2)).toEqualTypeOf<[3, 4, 5]>();
```

Comment:
when dropping the "const" from the function generics this would become
```suggestion
    expectTypeOf(drop([1, 2, 3, 4, 5], 2)).toEqualTypeOf<[number, number, number]>();
```

Reply:
Yup
Done